QuoteVerification/dcap_quoteverify/sgx-dcap-quoteverify-rs/src/lib.rs

```diff
@@ -28,24 +28,69 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  */
-//! Intel(R) Software Guard Extensions Data Center Attestation Primitives (Intel(R) SGX DCAP)
+//! Intel®️ Software Guard Extensions Data Center Attestation Primitives (Intel®️ SGX DCAP)\
 //! Rust wrapper for Quote Verification Library
 //! ================================================
 //!
 //! This is a safe wrapper for **sgx-dcap-quoteverify-sys**.
 
+#[cfg(test)]
+mod test;
+
```

Comment:
Maybe you forgot to add these tests.

Reply:
Mis added. We don't have test on this repo.